ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -116,7 +138,23 @@ export const OverlayClosedGroup = () => {
         disabled={noContactsForClosedGroup}
         onClick={onEnterPressed}
         dataTestId="next-button"
+        margin="auto 0 var(--margins-lg) 0 " // just to keep that button at the bottom of the overlay (even with an empty list)
       />
     </div>
   );
 };
+
+const StyledMemberListNoContacts = styled.div`
```

Comment:
Similar thing to above where Styled Compoents should be above the main rendered component. Do we care?